### ISSUE TITLE
Incorrectly shows 100%

### ISSUE THREAD

**Issue by Avamander** [jfurrow/flood#558]
Flood incorrectly displays "Downloaded | 100%" but rtorrent reports otherwise (99,98% complete of a 100GB torrent). It would be nice if it didn't do any rounding in the precentage section in advanced details, having more decimal places wouldn't hurt anyone but would avoid all situations like these.

**Comment by noraj1337** [comment]
Yes, and when no in the advanced details it's better to truncate `99,98%` into `99%` than round it up to `100%`.

**Comment by Syc0x00** [comment]
@jfurrow I believe this issue is complete.
